### PR TITLE
Completed javascript-data-model.

### PR DESCRIPTION
**Pull request by bookaneers**
css-data-model | Fernando Figueiredo
Screen Shot 2021-04-01 at 9 08 59 PM

### CODE REVIEW

File: javascript-data-model/order-history.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+var orderHistory = [
+  {
+    orderNumber: '001',
+    orderPlaced: '03 / 16 / 2021',
+    booksOrdered: [{
```
Comment (BrettAlbright) [inline javascript-data-model/order-history.js]: nice idea using a separate property for the different type of items! so the next step in that idea is to create a property named `items` which will contain an array of objects in which each object contains an individual items data: 
```javascript
var orderHis = [
  {
    orderedDate: '2020-08-04',
    totalPrice: 34,
    shipTo: 'JS Masher',
    orderNum: '114-3941689-8772232',
    deliveredDate: '2020-08-08',
    orderedDate: '2-15-21',
    items:[{
      type: "book",
      title: "A New Way to Learn"
    },
    {
      type: "electronics",
      title: "Blu Ray Player"
    },
    {
      type: "book",
      title: "How to forget"
    },
    {
      type: "book",
      title: "how to remember"
    }
  ]
  }
]
```

you will notice you already hit this idea, but this update will allow you have to any number of different item types without having to update the model between orders.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Good initial submission for the first order. Make sure to include the data for the other orders as well, and to make the requested update.

**Comment by bookaneers** [comment]
javascript-data-model  | Fernando Figueiredo
Screen Shot 2021-04-02 at 1 57 32 PM

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Structure is great! Now just make sure to added the remaining orders!

**Comment by bookaneers** [comment]
javascript-data-model | Fernando Figueiredo
Screen Shot 2021-04-02 at 3 35 34 PM

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**